src/main/java/com/gachigang/ontherun/persistence/entity/Category.java

```diff
@@ -32,4 +32,9 @@ public class Category {
 
     @OneToMany(mappedBy = "category", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
     private List<Product> products;
+
+    @ElementCollection
+    @CollectionTable(name = "category_product_id", joinColumns = @JoinColumn(name = "category_id"))
+    @Column(name = "product_id")
+    private List<Long> productId;
```

Comment:
Remove it, we cannot map List to single field

Reply:
Done